content/en/docs/orientation/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "New Contributor Orientation Sessions"
+weight: 3
+description: |
+  The community conducts monthly online sessions to orient new Kubernetes
+  contributors.
+---
+
+If you're new to the Kubernetes community and looking for a place to contribute,
+it can be challenging to figure out how to get started.  New contributors should
+join us for one of the New Contributor Orientation sessions, held once a month
+on Zoom.
+
+**Meeting Time**: every 3rd Tuesday of the month for 1 hour
+
+<a href="https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NXVpdGhoMWRyMGhpMDZjd[…]afef04s12ra0gkqql6fchjc%40group.calendar.google.com&scp=ALL" >EMEA/APAC-friendly: 1:30 PT / 8:30 UTC / 10:30 CET / 14:00 IST</a>
+
+<a href="https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MnZqMXVmazZhNWJ2aTNld[…]afef04s12ra0gkqql6fchjc%40group.calendar.google.com&scp=ALL" >AMER-friendly: 8:30 PT / 15:30 UTC / 17:30 CET / 21:00 IST</a>
```

Comment:
These hyperlinks look truncated. Otherwise, LGTM.

Reply:
Feh, @mfahlandt ?  Those are your links.

Reply:
Solved this by removing the links entirely.